sources/packages/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -116,32 +129,149 @@ export class AssessmentControllerService {
       );
     }
 
-    //Disbursement data is populated with dynamic key in a defined pattern to be compatible with form table.
+    return {
+      assessment: assessment.assessmentData,
+      applicationNumber: assessment.application.applicationNumber,
+      fullName: getUserFullName(assessment.application.student.user),
+      programName: assessment.offering.educationProgram.name,
+      locationName: assessment.offering.institutionLocation.name,
+      offeringIntensity: assessment.offering.offeringIntensity,
+      offeringStudyStartDate: getDateOnlyFormat(
+        assessment.offering.studyStartDate,
+      ),
+      offeringStudyEndDate: getDateOnlyFormat(assessment.offering.studyEndDate),
+      msfaaNumber: assessment.application.msfaaNumber.msfaaNumber,
+      disbursement: this.populateDisbursementAwardValues(
+        assessment.disbursementSchedules,
+      ),
+    };
+  }
+
+  /**
+   * Disbursement data is populated with dynamic key in a defined pattern to be compatible with form table.
+   * @param disbursementSchedules disbursement schedule details.
+   * @param includeDocumentNumber when true document number is mapped
+   * to disbursement dynamic data.
+   * @returns disbursement dynamic award data.
+   */
+  private populateDisbursementAwardValues(
+    disbursementSchedules: DisbursementSchedule[],
+    includeDocumentNumber?: boolean,
```

Comment:
This would be a boolean variable IMO. I would prefer to have it declared as optional with default to `false` `includeDocumentNumber = false`.